kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -83,6 +85,14 @@ edgeAgent:
     # literal "-" in liue of an empty string.
     # storageClassName: ""
 
+    # Set this to true if you wish to turn on experimental features support
+    # enableExperimentalFeatures: true
+    
+    # Set this to true if you wish to turn on Kubernetes create options support.
+    # In order to fully enable this experimental options for work you should
+    # set "true" both parameters "enableExperimentalFeatures" and "enableK8sExtensions"
+    # enableK8sExtensions: true
```

Comment:
We should provide the template for the experimental features we support here, so it is documented somewhere closer to the code.

Reply:
Do you mean a link to https://github.com/Azure/iotedge/blob/master/kubernetes/doc/create-options.md ?

Reply:
Added a link